### PR TITLE
Add string tensor support to memory_alloc

### PR DESCRIPTION
**Pull request by CoderHam**
- [x] Test String tensors in L0_io

### CODE REVIEW

**Comment by deadeyegoodwin** [comment]
@CoderHam The CI seems to have failed. Can you rebase this to master, fix any issue, and try again.

File: qa/L0_io/test.sh [hunk 1]
```diff
@@ -78,7 +78,36 @@ for trial in graphdef savedmodel netdef onnx libtorch plan ; do
     fi
 done
 
-# custom model needs to be obtained elsewhere
+# Prepare object models with basic config
```
Comment (deadeyegoodwin) [inline qa/L0_io/test.sh]: object -> string

File: qa/L0_io/test.sh [hunk 2]
```diff
@@ -78,7 +78,36 @@ for trial in graphdef savedmodel netdef onnx libtorch plan ; do
...
+                sed -i "s/label_filename:.*//" config.pbtxt && \
+                echo "instance_group [{ kind: KIND_CPU }]" >> config.pbtxt)
+
+    # ensemble version of the model.
+    mkdir -p $MODELSDIR/fan_${full}/1 && \
+    cp $ENSEMBLEDIR/fan_${full}/config.pbtxt $MODELSDIR/fan_${full}/. && \
+        (cd $MODELSDIR/fan_${full} && \
+                sed -i "s/label_filename:.*//" config.pbtxt)
+
+    if [ "$trial" == "libtorch" ]; then
```
Comment (GuanLuo) [inline qa/L0_io/test.sh]: Are you planning on adding libtorch string model?

Comment (CoderHam) [reply qa/L0_io/test.sh]: Oops, removing this, libtorch does not support string

**Pull request merged**